primitives/src/common.rs

```diff
@@ -0,0 +1,14 @@
+use rstd::vec::Vec;
+
+/// Substrate runtimes provide no string type. Hence, for arbitrary data of varying length the
+/// `Vec<u8>` is used. In the polkadot-js the typedef `Text` is used to automatically
+/// utf8 decode bytes into a string.
+pub type PalletString = Vec<u8>;
```

Comment:
why rename the original type?

Reply:
I wanted to make it semantically clear that the data underlying the `Vec<u8>` is an utf8 encoded string in cases where it is not obvious.

For instance, we previously had the field `name_utf8` in the `CommunityProperties`. Here we could safely remove the suffix without potentially causing a misunderstanding.

I believe, trying to be as explicit as possible gives better understanding when reading the code.